packages/react-components/src/components/MessageThread.test.tsx

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT license.
 
 import React from 'react';
+/* @conditional-compile-remove(mention) */
+import { v1 as generateGUID } from 'uuid';

```

Comment:
Just a general thing with respect to UUIDs. v1 is fine for test / simple scenarios but if this were to be used where uniqueness is truly needed, v4 or v5 should be used.

Reply:
Will keep it as v1 uuid since it's being used in several other places in the repo already and should be sufficient for testing purpose like you said, but thanks for sharing more details regarding v4/v5 UUID being unique